src/UpdatePassword.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import {withRouter, Link} from 'react-router-dom'
+import {getCookie} from './cookies';
+import Header from './Header/Header'
+
+
+class UpdatePassword extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        userId: getCookie("userId"),
+        oldPassword: "",
+        newPassword:"",
+        confirmPassword:"",
+        error: "",
+      };
+      
+      this.handleInputChange = this.handleInputChange.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+      
+    }
+    
+    handleInputChange(event) {
+        const target = event.target;
+        const value = target.value;
+        const name = target.name;
+
+        this.setState({
+            error: "",
+          [name]: value,
+        });
+
+    }
+
+    completedInput = () => {
+        const inputs = ['oldPassword', 'newPassword', 'confirmPassword'];
+        for(var i =0; i <inputs.length; i++) {
+          if(!this.state[inputs[i]])
+            return false;
+        }
+        return true;
+    }
+
+    checkPasswords() {
+        const newPassword = this.state.newPassword;
+        const confirmPassword = this.state.confirmPassword;
+        const oldPassword = this.state.oldPassword;
+        if (newPassword.length === 0) {
+          alert("Please add password");
+          return false;
+        } else if (confirmPassword != newPassword) {
+          alert("Please make sure the new passwords match");
+            
+          return false;
+        } else if ((confirmPassword == newPassword) && (newPassword == oldPassword)){
+            alert("Your new password is the same as your old password")
+            return false;
+        }
+        return true;
+    }
+        
+    handleSubmit = (event) => {
+        event.preventDefault();
+        
+        if (this.completedInput()){
+            if (this.checkPasswords()){
+                const id = this.state.userId;
+                const myForm = new FormData (document.getElementById("newPasswordForm"));
+                myForm.append("id", id);
+
+                const myRequest = new Request("http://127.0.0.1:5000/updatePassword", {
+                    method: "POST",
+                    body: myForm,
+                });
+  
+                fetch(myRequest)
+                .then((res) =>
+                res.json())
+                .then((res) => {
+                    this.setState({error: ""});
+                    if(res.response === "Success"){
+                        this.setState({
+                            error: "Successfully updated password!",
```

Comment:
This should not be in the error field

Reply:
it's in error field so that the success message and error message all shows up in the same place, or else due to cookie it will not be refreshed

Reply:
can you make another field for this?